src/pages/academy/game/Game.tsx

```diff
@@ -50,9 +51,82 @@ function Game() {
     }
   }, [session, achievements, goals]);
 
+  // Logic for the fullscreen button to toggle fullscreen.
+  // Entering fullscreen will also lock the screen orientation to landscape.
+  const [isFullScreen, setIsFullScreen] = React.useState(false);
+  const toggleFullScreen = () => {
+    const elem = document.getElementById('game-display') as HTMLElement;
+
+    if (!isFullScreen) {
+      if (elem.requestFullscreen) {
```

Comment:
Here instead of checking if the function exists on the element, we should instead check whether the page is currently full screen or not (refer to [here (MDN)](https://developer.mozilla.org/en-US/docs/Web/API/Document/fullscreenElement) for more details)